app/javascript/app/components/section/section-component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+import cx from 'classnames';
+
+import layout from 'styles/layout.scss';
+import styles from './section-styles.scss';
+
+const Section = ({ children, backgroundImage, className }) =>
+  (<section
+    className={styles.section}
+    style={{
+      backgroundImage: `url(${backgroundImage})`
+    }}
+  >
+    <div className={cx(className, layout.content, styles.doubleFold)}>
```

Comment:
To be able to override the default styles I'd move the className to the end of the list.